x-pack/plugins/secops/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "author": "Elastic",
+  "name": "sec-ops",
+  "version": "7.0.0-alpha1",
+  "scripts": {
+    "build-graphql-types": "node scripts/generate_types_from_graphql.js"
+  },
+  "devDependencies": {
+    "@types/boom": "7.2.0",
```

Comment:
I don't know what the philosophies of the `package.json`'s are yet with regards to using

`^7.2.0` vs `~7.2.0`, vs. `7.2.0` within their package.json's

but in their regular `package.json` they have all their `@types/xyz` with a carrot including this one so recommending being consistent and having:

```
"@types/boom": "^7.2.0",
```
here to match upstream `package.json`

or if you can remove it altogether since the main `package.json` has it already
https://github.com/elastic/kibana/blob/master/package.json#L229

and there is hopefully a cascade down effect with deps (which I think there is)